ui/helpers/utils/i18n-helper.tsx

```diff
@@ -110,19 +115,21 @@ export async function fetchLocale(localeCode) {
 
 const relativeTimeFormatLocaleData = new Set();
 
-export async function loadRelativeTimeFormatLocaleData(localeCode) {
+export async function loadRelativeTimeFormatLocaleData(
+  localeCode: string,
+): Promise<any> {
```

Comment:
Promise<any> does satisfy Promise<void> but in this case lets use the latter because we don't return anything from this method.

Reply:
I edited the above to put code ticks around the types because github tries to render html tags and `<void>` is one of those i guess lol

Reply:
:) hahah github is too smart